src/pages/login.jsx

```diff
@@ -37,7 +37,7 @@ export default function Login() {
     }
 
     dispatch(cleanFlash());
-  }, [errors, message, dispatch]);
+  }, [errors, message, dispatch, navigate]);
 
   return (
     <div className="h-100 d-flex flex-column align-items-center justify-content-center">
```

Comment:
I think you should handle the width of the form on medium screens, for a good user experience.
![Screenshot 2023-04-10 at 10 44 26 AM](https://user-images.githubusercontent.com/105528809/230877630-dbaf5882-d390-47bd-9b3e-1a362164f4f6.png)